### ISSUE TITLE
Adresses

### ISSUE THREAD

**Issue by MancusNemo** [PadWorld-Entertainment/worldofpadman#276]
What is going on here?
The Adress in the ingamemenue is wirtten with worldofpadman.com and in the browser i enter worldofpadman.net.
Is this historical? Is it a mistake? Is it art? I don't know.
For me it looks like a mistake and shoud be fixed.
First Picture is uncommented.
Second Picture is showing by red color the important places.
![Bildschirmfoto vom 2024-06-27 10-37-34](https://github.com/PadWorld-Entertainment/worldofpadman/assets/73421306/f25c384b-de2c-4555-bb6a-cc1ad2a8346f)
![Bildschirmfoto vom 2024-06-27 10-37-34 (Kopie)](https://github.com/PadWorld-Entertainment/worldofpadman/assets/73421306/25a94daf-209d-4b3e-870a-ed55ec8655a9)

**Comment by kai-li-wop** [comment]
Hi, worldofpadman.com is historically the correct address, registered and owned by ENTE since PadMod times. To support the game's web presence, I also registered worldofpadman.net and worldofpadman.org, which used to redirect to worldofpadman.com.

When the website crashed in 2013 (as I recall) and the team struggled for years to get it back online, I created a new website for the game in 2016 with ENTE's approval. To make it easier to maintain, I put it on my own web space at worldofpadman.net. ENTE then redirected from worldofpadman.com to worldofpadman.net.

So worldofpadman.com is still the official domain and will remain so forever, while the website currently runs on worldofpadman.net. I gave up worldofpadman.org years ago for cost reasons.
